app/src/layouts/common/sidebar/userBlock/userBlock.jsx

```diff
@@ -35,21 +35,24 @@ import styles from './userBlock.scss';
 
 const cx = classNames.bind(styles);
 
-const UserAvatar = ({ photoTimeStamp }) => (
+const UserAvatar = ({ photoTimeStamp, preloaderColor }) => (
   <div className={cx('avatar-wrapper')}>
     <Image
       className={cx('avatar')}
       src={URLS.dataPhoto(photoTimeStamp, true)}
       alt="avatar"
       fallback={DefaultUserImage}
+      preloaderColor={preloaderColor}
```

Comment:
Could you please explain why you decided to work with color through a parent if this method is not used anywhere else?

Reply:
`UserAvatar` uses in sidebar with tooltip we need here a white color preloader. Also, `UserAvatar` uses in pop-up by clicking on user avatar and here we need a charcoal color. That reason why I have needed lifting this props.

Reply:
Duration block, preloader on admin panel for all users image, attachments with photos, issue tooltip are having a charcoal color.
UserAvatar has white color.